### DISCUSSION TITLE
Expand The Auto Idle On DND/Idle!

### DISCUSSION THREAD

**Discussion by mishameinan** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#19] [Ideas]
I don't really know that would make the Plugin to be more complicated, but it would be great if the Auto Idle works either way no matter what your status is (except for invisible I guess). (I don't usually use the Online status that's why I'm asking if that would be possible lol)

**Comment by RoguedBear** [comment]
Currently the only way to change the trigger status would be my modifying the plugin code yourself and replacing hardcoded "online" strings with "dnd".

Though, during my summer break I am planning to overhaul the plugin entirely and add all the features labelled under [enhancement](https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
